### PR TITLE
Create the tree files and the very first version of the puller

### PR DESCRIPTION
**Pull request by brichet**
This PR is to initialize the repo.
It includes a very first version of the git puller using the Github API.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@jupyterlite/litegitpuller",
+  "version": "0.0.1-dev.0",
```
Comment (jtpio) [inline package.json:3]: Maybe we start with `0.0.1`, and we make a first `0.1.0` with the releaser?

Comment (jtpio) [reply package.json:3]:
```suggestion
  "version": "0.0.1",
```

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+build_cmd = "build:prod"
+npm = [
+  "jlpm",
+]
+
+# autoflake is used for autoformatting Python code
+#
+# ref: https://github.com/PyCQA/autoflake#readme
+#
+[tool.autoflake]
```
[the remark this answers is not in the document]
Comment (jtpio) [reply pyproject.toml:69]: Actually we might not even need any of these tools since it's a pure frontend extension.

**Comment by brichet** [comment]
Thanks @jtpio for the quick review, I should have start using a template.
I close this PR in favor of https://github.com/jupyterlite/litegitpuller/pull/2.

**Pull request closed without merging**